config/jobs/kubernetes/sig-release/release-branch-jobs/dra-1.32.yaml

```diff
@@ -72,163 +72,3 @@ periodics:
           memory: 6Gi
       securityContext:
         privileged: true
-- annotations:
-    testgrid-alert-email: eduard.bartosh@intel.com, patrick.ohly@intel.com
-    testgrid-dashboards: sig-node-dynamic-resource-allocation, sig-node-cri-o, sig-release-1.32-informing
-  cluster: k8s-infra-prow-build
-  decorate: true
-  decoration_config:
-    timeout: 1h30m0s
-  extra_refs:
-  - base_ref: release-1.32
-    org: kubernetes
-    path_alias: k8s.io/kubernetes
-    repo: kubernetes
-    workdir: true
-  - base_ref: master
-    org: kubernetes
-    path_alias: k8s.io/test-infra
-    repo: test-infra
-  interval: 24h
-  labels:
-    preset-k8s-ssh: "true"
-    preset-service-account: "true"
-  name: ci-node-e2e-crio-cgrpv1-dra-1-32
-  spec:
-    containers:
-    - args:
-      - kubetest2
-      - noop
-      - --test=node
-      - --
-      - --repo-root=.
-      - --gcp-zone=us-west1-b
-      - --parallelism=1
-      - '--label-filter=Feature: containsAny DynamicResourceAllocation && Feature: isSubsetOf { Beta, DynamicResourceAllocation } && !Flaky && !Slow'
-      - --timeout=60m
-      - --skip-regex=
-      - '--test-args=--feature-gates="DynamicResourceAllocation=true" --service-feature-gates="DynamicResourceAllocation=true" --runtime-config=api/beta=true --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
-      - --image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/latest/image-config-cgroupv1-serial.yaml
-      command:
-      - runner.sh
-      env:
-      - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
-        value: "1"
-      - name: GOPATH
-        value: /go
-      - name: KUBE_SSH_USER
-        value: core
-      image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250212-16f67660c2-1.32
```

Comment:
I'm checking whether kubetest2 in this image can be updated, but in the meantime let's revert to avoid red jobs in the release informing dashboard.